Frontend/src/components/entry/EntryForm.jsx

```diff
@@ -1,7 +1,21 @@
+import React from 'react';
 import AnnotationForm from "../annotation/AnnotationForm.jsx";
 import { getCurrentDate } from "../../utils/dateUtils";
 
 const EntryForm = ({ solutionIndex, entry, entryIndex, solutions, setSolutions }) => {
+
+  const initializeEntryDate = () => {
+    if (!entry.entry_date) {
+      const updatedSolutions = [...solutions];
+      updatedSolutions[solutionIndex].entries[entryIndex].entry_date = getCurrentDate();
+      setSolutions(updatedSolutions);
+    }
+  };
+
+  React.useEffect(() => {
+    initializeEntryDate();
+  }, []);
```

Comment:
Pedes revisar las dependencias?

```suggestion
  }, [entry, solutions, entryIndex, solutionIndex, setSolutions]);
```